operations/research-and-development/product/release-process/incident-collaboration-release.md

```diff
@@ -1,26 +1,26 @@
-# Incident Collaboration Release Process
+# Playbooks Release Process
 
-This document outlines the Incident Collaboration release process for the Mattermost core team. Our release process draws from many best practice Agile, Scrum, and Software Development Lifecycle methodologies.
+This document outlines the Playbooks release process for the Mattermost core team. Our release process draws from many best practice Agile, Scrum, and Software Development Lifecycle methodologies.
 
 Every two weeks, the team runs through a playbook to cut a new release of the plugin, smoke test it, and add it to the marketplace. That gets pushed to cloud immediately \(via split.io\), and any self-managed customers who desire the upgrade may do so from the plugin marketplace.
 
 ## Release Timeline
 
-Schedule for Incident Collaboration releases:
+Schedule for Playbooks releases:
 
 * Features and bug fixes are merged
 * \(T-3\): Cut a Release Candidate
 * \(T-2\): Smoketesting
 * \(T-0\): Release Day
 
-On the first Monday of each month, or the next business day, the Incident Collaboration team's Product Manager and Dev Lead choose what version of the plugin they will pre-package into that month's self-managed release. The deadline to merge the selected version into the self-managed release is the Code Freeze day. An additional round of testing is required to qualify before merging.
+On the first Monday of each month, or the next business day, the Playbooks team's Product Manager and Dev Lead choose what version of the plugin they will pre-package into that month's self-managed release. The deadline to merge the selected version into the self-managed release is the Code Freeze day. An additional round of testing is required to qualify before merging.
 
 ## Notes
 
 * Critical bugs are flagged and fixed before releasing, with quick dot releases completed as needed.
-* The team uses Incident Collaboration Playbook checklists for releases and for pre-packaging with self-managed.
-  * Playbooks Team: [Plugin Release playbook](https://community.mattermost.com/core/com.mattermost.plugin-incident-management/playbooks/hzgiqpzsbinpujdnue9xa1kj4y) \(has limited access\)
-  * Playbooks Team: [Feature Swimlane playbook](https://community.mattermost.com/core/com.mattermost.plugin-incident-management/playbooks/1gtdk5q57irzib67w6ocaatimy) \(has limited access\)
+* The team uses a ``Playbooks Team: Plugin Release`` checklist for releases and for pre-packaging with self-managed.
+  * Playbooks Team [Plugin Release playbook](https://community.mattermost.com/playbooks/playbooks/hzgiqpzsbinpujdnue9xa1kj4y/preview) \(has limited access\)
+  * Playbooks Team [Feature Swimlane playbook](https://community-daily.mattermost.com/playbooks/playbooks/1gtdk5q57irzib67w6ocaatimy/preview) \(has limited access\)
```

Comment:
Question: The Plugin Release and the Feature Swimlane playbook links point to community and community-daily, respectively. Is that intentional? I ask because we generally provide links to community over community-daily when the content is publicly facing.

Reply:
Thank you! Fixed it!